srml/elections-phragmen/src/lib.rs

```diff
@@ -159,6 +162,11 @@ decl_storage! {
 		/// The present candidate list. Sorted based on account id. A current member can never enter
 		/// this vector and is always implicitly assumed to be a candidate.
 		pub Candidates get(fn candidates): Vec<T::AccountId>;
+
+		/// Has the storage format been updated?
+		/// NOTE: Only use and set to false if you have used an early version of this module. Should
+		/// be set to true otherwise.
+		pub DidUpdate: bool = false;
```

Comment:
DidMigrate? Since DidUpdate has already been used for in a different context (e.g. see `timestamp`)